bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/providers/cli/CliProvider.java

```diff
@@ -172,18 +172,30 @@ public boolean hasConfigs() {
 
     @Override
     public Optional<BArray> getAsArrayAndMark(Module module, VariableKey key) {
+        CliArg cliArg = getCliArg(module, key);
```

Comment:
As the same lines are executed for methods `getAsArrayAndMark`, `getAsRecordAndMark` & `getAsTableAndMark` , Shall we move it to a separate method and call?

Reply:
In other get methods also same. Only in this case we throw a not support error until supported and other places we throw a different error. Shall we keep this since its more clear for me?